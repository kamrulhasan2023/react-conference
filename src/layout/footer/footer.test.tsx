import { render } from '@testing-library/react';
import Footer from '.';

it('render footer correctly', () => {
  const { getAllByRole } = render(<Footer />);
  const element = getAllByRole('presentation')[0];

  expect(element).toBeInTheDocument();
  expect(element).toMatchSnapshot();
});
