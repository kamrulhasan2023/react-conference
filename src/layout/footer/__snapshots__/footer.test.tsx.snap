// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render footer correctly 1`] = `
<footer
  class="bg-black-light py-6 lg:py-8"
  role="presentation"
>
  <div
    class="container"
    role="presentation"
  >
    <p
      class="text-center text-base text-white"
    >
      Â© Lemon Hive. 
      2023
    </p>
  </div>
</footer>
`;
